outlier_screening.Rmd

```diff
@@ -1,37 +1,87 @@
----
-title: "Outlier Screening"
-author: "Emily Linebarger"
-date: "2/17/2021"
-output: pdf_document
----
-
-```{r setup, include=FALSE}
-knitr::opts_chunk$set(echo = FALSE)
-library(data.table)
-
-# Set working directory to root of repository 
-stopifnot(grepl("data_557", getwd()))
-dt = fread("prepped_data/sixmonth.csv")
-```
-
-Run first regression, of just cases-per-capita to overall score. 
-```{r reg1} 
-summary(lm(casepc ~ overall, data=dt))
-plot(lm(casepc ~ overall, data=dt))
-```
-
-Do a little bit of outlier screening, and then rerun regression. 
-```{r outlier_screening} 
-range = IQR(dt$casepc, na.rm=TRUE)
-mean = mean(dt$casepc, na.rm=TRUE)
-subset1 = dt[casepc<mean-range | casepc>mean+range]
-
-dt = dt[!country_code %in% subset1$country_code]
-
-```
-
-
-```{r reg2}
-summary(lm(casepc ~ overall, data=dt))
-plot(lm(casepc ~ overall, data=dt))
-```
+---
+title: "Outlier Screening"
+author: "Emily Linebarger"
+date: "2/17/2021"
+output: pdf_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+library(data.table)
+library(knitr)
+
+# Set working directory to root of repository 
+stopifnot(grepl("data_557", getwd()))
+dt = fread("prepped_data/sixmonth.csv")
+```
+
+Run first regression, of just cases-per-capita to overall score. 
+```{r reg1} 
+summary(lm(cfratio ~ overall, data=dt))
+plot(lm(cfratio ~ overall, data=dt))
+```
+
+```{r hist} 
+hist(dt$cfratio)
+```
+
+There seems to be constant variance in plots 1 and 3. However, we have an issue with non-normality in our qq-plot. 
+Examining this further with a histogram, we can see two issues: a large cluster of zeros, and a few high outliers with a case fatality ratio higher than 25. 
+
+```{r boxplot} 
+boxplot(dt$cfratio)
+summary(dt$cfratio)
+upper_whisker = 3.327 + ((3.327-1.080) * 1.5)
```

Comment:
Where did you get the value of 1.5 from? I feel like I've seen cutoffs like 2 or 3 times the standard deviation from the mean. Curious if that changes the results.

Reply:
I was reading this documentation: https://www.rdocumentation.org/packages/graphics/versions/3.6.2/topics/boxplot The "range" argument gives you the extension past the interquartile range, which is by default 1.5. (I was reading a couple of blogs that also said this was the correct interpretation). We could definitely toy with higher values like 2, 3 etc.